util.sh

```diff
@@ -47,6 +47,28 @@ function build_plist() {
     fi
 }
 
+################################################################################
+# Build font instances from designspace and UFOs.
+# Arguments:
+#     Path to designspace source.
+#     Output formats, as separate strings.
+################################################################################
+function build_designspace() {
+    filename=$(basename "$1"); filename="${filename%.*}"
+    fontmake -m "$1" -i -o "${@:2}" --expand-features-to-instances
+    for format in "${@:2}" ; do cp master_$format/$filename-*.$format instance_$format/; done
```

Comment:
please remove this line

Reply:
with -i, fontmake will generate fonts in instance_ttf and instance_otf directories